src/jit/codegencommon.cpp

```diff
@@ -4835,11 +4835,12 @@ void CodeGen::genCheckUseBlockInit()
         }
 
 #ifdef _TARGET_XARCH_
-        // If we're going to use "REP STOS", remember that we will trash EDI
-        // For fastcall we will have to save ECX, EAX
-        // so reserve two extra callee saved
-        // This is better than pushing eax, ecx, because we in the later
-        // we will mess up already computed offsets on the stack (for ESP frames)
+        /* If we're going to use "REP STOS", remember that we will trash EDI */
+        /* For fastcall we will have to save ECX, EAX
+         * so reserve two extra callee saved
+         * This is better than pushing eax, ecx, because we in the later
+         * we will mess up already computed offsets on the stack (for ESP frames)
+         */
```

Comment:
I am up to date. Thanks! Long story... Had to revert so, I branch this branch as close as the master for the next work item. I will make sure I check in the right style comment.